dtm/remote.c

```diff
@@ -60,7 +60,8 @@ static struct m0_fop_type rem_rpc_fopt;
 static const struct m0_fop_type_ops rem_rpc_ftype_ops;
 static struct m0_rpc_item_type_ops rem_rpc_itype_ops;
 static const struct m0_rpc_item_ops rem_rpc_item_sender_ops;
-static const struct m0_rpc_item_ops rem_rpc_item_redo_ops;
+/* unused
```

Comment:
Do we need to delete all the unused code?

Reply:
Well, maybe there were some future plans on this code...